be/src/column/array_column.cpp

```diff
@@ -382,6 +382,16 @@ int ArrayColumn::compare_at(size_t left, size_t right, const Column& right_colum
     return lhs_size < rhs_size ? -1 : (lhs_size == rhs_size ? 0 : 1);
 }
 
+void ArrayColumn::compare_column(const Column& rhs_column, std::vector<int8_t>* output) const {
+    CHECK(size() == rhs_column.size()) << "Two input columns must have same rows";
```

Comment:
why we have this limitation?what will happen if we compare two array with different length?